DESCRIPTION

```diff
@@ -42,6 +42,7 @@ Imports:
     rjags,
     survival,
     tibble,
+    tidyverse,
```

Comment:
can we move this to `Suggests` ? if only used in vignettes.

Reply:
I have removed all references to the `tidyverse` metapackage entirely.  Vignettes now 
- Reference individual packages (eg `dplyr`) directly
- Use `requireNamespace()` to see if the necessary packages are installed and, if not, fail elegantly.